packages/rancher-istio/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -4,15 +4,50 @@ spec:
   components:
     base:
       enabled: {{ .Values.base.enabled }}
+      k8s:
```

Comment:
all of these components require nodeSelectors and tolerations, which can be supplied if you look at the IstioOperator spec: https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#IstioOperatorSpec

Currently, there were no nodeSelectors or tolerations